src/pages/settings/Wallet/ChooseTransferAccountPage.tsx

```diff
@@ -50,40 +52,73 @@ function ChooseTransferAccountPage({walletTransfer = {}}: ChooseTransferAccountP
         BankAccounts.openPersonalBankAccountSetupView();
     };
 
+    const [bankAccountsList] = useOnyx(ONYXKEYS.BANK_ACCOUNT_LIST);
+    const selectedAccountID = walletTransfer?.selectedAccountID;
+    const data = useMemo(() => {
+        const options = Object.values(bankAccountsList ?? {}).map((bankAccount) => {
```

Comment:
Why don't you filter bank accounts like the previous logic in PaymentMethodList anymore?